spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -218,6 +225,28 @@
       release_id = api_client.upload("app_id", fake_binary_path)
       expect(release_id).to eq("release_id")
     end
+
```

Comment:
Consider adding a test for the scenario where the initial call to get the upload_status returns a response with status equal to `IN_PROGRESS`. For this, the test would make sure that upload is **not** called.